packages/react-devtools-shared/src/backend/views/TraceUpdates/canvas.js

```diff
@@ -79,7 +95,11 @@ function drawBorder(
   context.setLineDash([0]);
 }
 
-export function destroy(): void {
+export function destroy(agent: Agent): void {
+  if (window.document == null) {
+    return;
```

Comment:
Are we going to implement destroy for native?

Reply:
I thought I don't need this, but with the current implementation (where I don't need to insert a native module), it actually makes sense to unsubscribe and not render the overlay when the highlight toggle is off. I'll make changes here.